app/models/item.rb

```diff
@@ -19,15 +16,13 @@ def self.sql_injection(params)
     search_terms = params.permit(attributes.keys).to_h
     injection = search_terms.map do |k,v|
       param_generator(k, v)
-      # "lower(#{key}) LIKE '%#{value.downcase}%'"
     end
     injection.join(" AND ")
   end
 
   def self.param_generator(key, value)
     if key.include?('ated_at')
-      value = value.to_datetime
-      "#{key} >= '#{value}' AND #{key} < '#{value+1.seconds}'"
+      "to_char(#{key}, 'YYYY-MM-DD HH24:MI:SS UTC +00:00') LIKE '%#{value}%'"
```

Comment:
This is a more useful search. now it will return if the search value is included in any part of the date